### PR TITLE
license-status

### PR DESCRIPTION
**Pull request by sharmayajush**
- get license status 

signed-off-by: Yajush Sharma <yajush@accuknox.com>

### CODE REVIEW

**Review by rajaSahil — APPROVED** [review]

File: cmd/get/licenseStatus.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+		}
+		defer conn.Close()
+		licenseClient := pb.NewLicenseClient(conn)
+		req := &pb.LicenseStatusRequest{}
+		resp, err := licenseClient.GetLicenseStatus(context.Background(), req)
+		if err != nil {
+			fmt.Printf("unable to get license status: %s\n", err.Error())
+			return err
+		}
+		fmt.Println(resp)
```
Comment (rajaSahil) [inline cmd/get/licenseStatus.go]: Can you format the output?

Comment (sharmayajush) [reply cmd/get/licenseStatus.go]: Printed all the fields which are there in resp.

**Comment by rajaSahil** [comment]
@sharmayajush This is how output is being shown:
```
key:"U2FsdGVkX18SfYrxPZbvXS+tCe2ODFPhfpjLgQ85d87/wfIJEZPNmjnYd6n7X7P+JnYCDNZlIs3zWuwHsY414E6m6cxgGnWGTK2UWLE/luvMOMhw9jxgWmX+wd11Tbee1BUyuBzqolGSd34Yf6VLuRxIaf7uMIQrxLf+0kYneaLhHdqc0PkkTQ6d/kuYeKF60E/5PZhSZUCzojnvprPVvZdOaixV+WFlwS1JjxsG1Yt8KT3ymIvzD834Re+65VZzNDLaXPfo3gmF5I2oM4MZrQTaEt4SSm8xGZdMap8+SkkbByKb7wEzjzYwdQ1cg9V1cyJltFNtjuoMIRqrQ64N9LWCbzaF7/nf2G0aNhwSjF291GonSqZUQ8RzsgW4IEB9pYPNlLOZRvK1ShuZaHW3K+ZUiLQ+6Yikgal7B1eRvD+ZsdMb384XBLr3gOe79juSrSouNwi29sFXyquN+r5QAtOcWFX4r30kxjUDT/ayrZ8x6SZu6RjLjKZIPjgTJa1i3ABDEEf2IGNR25CgK3DltEParGNtfbSC3FIPfmo2LAVFi9VChBr6aGg2CdM59Fd4NokrhBuX5yqhgaIBRSKu7QEe1LSDS3aL5neHDEQ3rhI="  userId:"sahil@accuknox.com"  platformUUID:"1b676136-1a00-4396-9387-74889d236a3a"  issuedAt:"2023-04-27 08:01:07 +0000 UTC"  expiration:"2023-04-27 08:06:07 +0000 UTC"  features:"discover"  features:"summary"  features:"recommend"  status:"Active"
```

**Comment by rajaSahil** [comment]
@sharmayajush The output format should be
```
{
  "key": "U2FsdGVkX1/wybfYB/yjn8mjeDCeZ0fRiA1E3qHqHK0cUA4T9fbFfXp9n4nkT79D9oUBWJPQCr2w4CMC+bL9aH7250NBYWutQcNDviOSxYsExvwUYfA8GkAcYuyWcg+9KliYci3K34LKrpoyWxa611H2Ou2F2rOBgm5dS9KuAXd8dQ3yD0FdHxFS/6zYlaFN0pqsS9cuOKiRA6dQOQ07q0jhSItgdkbG/sQyVJcb6tEUvxPAETCmNe4fSTrz8USfPzn/Rvsjt1b+9i91F5VvI9wFqBYqWXD45QGOqg8HldRCeBkkauAagrkPzfGb9YrU+rfPtJ4sr4l8CVHlvfBIyfWawzoP0fZpT13s9NL1nZygFXx7m89nc6X4Y0LnBOtMDiS6isBi6/YsOfKhjq8UMIzY8btxhZmTGSN0i0vAsb/jxaRzH+kG6gIp72VfMcDdeZpVHCrdgDQKvOc/hAsxjSfKUMV5V4X5HNZJ7yfWZ9ST4mysJ4aXpnluq1M1ZPSu/rHc0D6qCBjPUy7NoQJnh5+fBKLMOqwdzNFR1bSptlkQAJuIL5RTjYaFy0DGDXi01wSrkHLRrB+N9zN3s7nc5MXrXAPxWkTC5okrSaKSZzI=",
  "userId": "sahil@accuknox.com",
  "platformUUID": "1b676136-1a00-4396-9387-74889d236a3a",
  "issuedAt": "2023-04-28 04:44:56 +0000 UTC",
  "expiration": "2023-05-05 04:44:56 +0000 UTC",
  "features": [
    "discover",
    "summary",
    "recommend"
  ],
  "status": "Active"
}
```

**Review by rajaSahil — APPROVED** [review]

**Pull request merged**